docs/Makefile

```diff
@@ -20,3 +20,4 @@ help:
 %: Makefile
 	@$(SPHINXAPIDOC) $(MISCDOCFLAGS) -s $(DOCOUTPUT) -o $(OUTPUTDIR) $(SRCPROJECT) $(IGNORE)
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+	git checkout master -- _rtd/landing.md
```

Comment:
what's this for?

Reply:
Ah yes. If you run `make html` locally, what happens is that the function to append `README.md` into `landing.md` actually directly changes `landing.md` on your end. So there are two ways you can handle this: either force yourself to change `landing.md` each time you run `make html`, or just have the `Makefile` restore the original version from `master` each time. I think the latter is simpler 😄.

Reply:
gotcha.  another possible approach is to have dedicated start/stop injection flags/lines where the README is injected into.  That way, you can find where the readme injection starts, delete content until the injection stop flag, and then inject the readme below the start injection line.  I do some of that in AMP's `parse_ui.py` (editing `main_viewer.py` and `background_removal.py`) if you wanna see an implementation example.

idk which is better, but using git in a makefile like this shivers my timbers.